### DISCUSSION TITLE
Datadog Integration

### DISCUSSION THREAD

**Discussion by solhosty** [planetscale/discussion#597] [Q&A] [answered]
Hey guys, 

I added the Datadog integration but found out shortly after that this is pretty much only metrics on the read/writes and connections. I am wondering what the best route is to stream data from certain tables to Datadog for a dashboard view there? 

I'm not able to find much documentation outside of the Datadog integration you have, so just wondering if there is a way I can still get this set up to Datadog. 

Thanks!

**Comment by orware** [comment, marked as answer]
Hello @solhosty,

What do you mean by the "stream data" part you shared above?

Do you actually want to query your database and display the data within a dashboard elsewhere?

The Datadog integration is more for some of the basic metrics you mentioned as [described in our documentation](https://planetscale.com/docs/integrations/datadog), but isn't intended to display any data from the actual tables.

If that's more along the lines of what you were wanting, you can use platforms like [Metabase](https://metabase.com/) to connect to a PlanetScale database remotely and run queries, or use our other integrations for [Airbyte](https://planetscale.com/docs/integrations/airbyte), [Fivetran](https://planetscale.com/docs/integrations/fivetran), etc. to setup a syncing process into an external analytics datastore, such as BigQuery, which you can then use to query and create dashboards from too.

**Reply by solhosty** [reply]
Thank you @orware! Yes exactly that, i'll take a look deeper into the solutions you recommended, but they already look like good fits just scumming docs.
